docs/tutorials/integrations/ghactions.md

```diff
@@ -0,0 +1,15 @@
+---
+template: main.html
+---
+
+# Use Iter8 in a GitHub workflow
```

Comment:
Could we please say: `Use Iter8 in a GitHub Actions workflow`

Reply:
done